gnome-extra/cinnamon/cinnamon-4.4.8.ebuild

```
@@ -0,0 +1,193 @@
+
+COMMON_DEPEND="${PYTHON_DEPS}
+	app-accessibility/at-spi2-atk:2
+	app-misc/ca-certificates
+	dev-libs/dbus-glib
+	>=dev-libs/glib-2.35.0:2[dbus]
+	>=dev-libs/gobject-introspection-1.29.15:=
+	>=dev-libs/json-glib-0.13.2
+	>=dev-libs/libcroco-0.6.2:0.6
+	dev-libs/libxml2:2
+	gnome-base/librsvg
+	>=gnome-extra/cinnamon-desktop-4.0:0=[introspection]
+	>=gnome-extra/cinnamon-menus-4.4
+	>=gnome-extra/cjs-4.0.0[cairo]
+	>=media-libs/clutter-1.10:1.0[introspection]
+	media-libs/cogl:1.0=[introspection]
+	>=gnome-base/gsettings-desktop-schemas-2.91.91
+	media-libs/gstreamer:1.0
+	media-libs/gst-plugins-base:1.0
+	net-libs/libsoup:2.4[introspection]
+	>=sys-auth/polkit-0.100[introspection]
+	x11-libs/gdk-pixbuf:2[introspection]
+	>=x11-libs/gtk+-3.12.0:3[introspection]
+	x11-libs/pango[introspection]
+	>=x11-libs/startup-notification-0.11
+	x11-libs/libX11
+	>=x11-libs/libXfixes-5.0
+	>=x11-wm/muffin-4.0.3[introspection]
+	dev-libs/keybinder:3[introspection]
+	>=x11-libs/libnotify-0.7.3:0=[introspection]
+"
+# Runtime-only deps are probably incomplete and approximate.
+# Each block:
+# 2. Introspection stuff + dconf needed via imports.gi.*
+# 3. gnome-session is needed for gnome-session-quit
+# 4. Control shell settings
+# 5. accountsservice is needed for GdmUserManager (0.6.14 needed for fast
+#    user switching with gdm-3.1.x)
+# 6. caribou needed for on-screen keyboard
+# 7. xdg-utils needed for xdg-open, used by extension tool
+# 8. imaging, lxml needed for cinnamon-settings
+# 9. gnome-icon-theme-symbolic needed for various icons
+# 10. pygobject needed for menu editor
+# 11. nemo - default file manager, tightly integrated with cinnamon
+# 12. polkit-gnome - explicitly autostarted by us
+# TODO(lxnay): fix error: libgnome-desktop/gnome-rr-labeler.h: No such file or directory
+RDEPEND="${COMMON_DEPEND}
+	>=gnome-base/dconf-0.4.1
+	>=gnome-base/libgnomekbd-2.91.4[introspection]
+	sys-power/upower[introspection]
+
+	>=gnome-extra/cinnamon-session-4.0
+	>=gnome-extra/cinnamon-settings-daemon-4.0
+
+	>=app-accessibility/caribou-0.3
+
+	dev-libs/libtimezonemap
+	x11-misc/xdg-utils
+	x11-libs/xapps[introspection]
+
+	$(python_gen_cond_dep '
+		dev-python/dbus-python[${PYTHON_USEDEP}]
+		dev-python/pygobject:3[${PYTHON_USEDEP}]
+		dev-python/pexpect[${PYTHON_USEDEP}]
+		dev-python/pycairo[${PYTHON_USEDEP}]
+		dev-python/pyinotify[${PYTHON_USEDEP}]
+		dev-python/pypam[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/setproctitle[${PYTHON_USEDEP}]
+		dev-python/tinycss[${PYTHON_USEDEP}]
+		dev-python/pytz[${PYTHON_USEDEP}]
+		dev-python/xapp[${PYTHON_USEDEP}]
+	')
+
+	x11-themes/gnome-themes-standard
+	x11-themes/adwaita-icon-theme
+
+	>=gnome-extra/nemo-4.0
+	>=gnome-extra/cinnamon-control-center-4.0[networkmanager=]
+	>=gnome-extra/cinnamon-screensaver-4.0
+
+	gnome-extra/polkit-gnome
+
+	nls? ( >=gnome-extra/cinnamon-translations-4.0 )
+"
+DEPEND="${COMMON_DEPEND}
+	dev-util/gtk-doc
+	>=dev-util/intltool-0.4
```

```suggestion
	>=dev-util/intltool-0.40
```
I think there is missing zero at the end: https://github.com/linuxmint/cinnamon/blob/master/configure.ac#L31